lib/Doctrine/DBAL/Types/ConversionException.php

```diff
@@ -36,14 +36,19 @@ class ConversionException extends \Doctrine\DBAL\DBALException
      *
      * @param string $value
      * @param string $toType
+     * @param \Exception|null $previous
      *
      * @return \Doctrine\DBAL\Types\ConversionException
      */
-    static public function conversionFailed($value, $toType)
+    static public function conversionFailed($value, $toType, \Exception $previous = null)
```

Comment:
This would work I guess. Feel free to close this PR if you feel this is feature creep, I would be happy enough with this kind of solution. I just saw something similar in this very class, and thought this would be ok.